app/assets/stylesheets/application.scss

```diff
@@ -1,98 +1,478 @@
+  border: 2px solid #000000;
+  box-shadow: -1px -1px 10px 10px #000000;
+}
+#city-show-button {
+  align-items: center;
+  display: block !important;
+  font-size: 24px;
+  color: #cb0000;
+  transition-duration: 0.3s;
+}
+#city-show-button:hover {
+  color: #42bfbf !important;
+}
+#my-search-h3 {
+  padding-top: 20vh;
+  color: #ffffff;
+}
+#search-index-page {
+  min-height: 100vh;
+  background: linear-gradient(45deg, #d3d3cd 50%, #000000 50%);
+}
+#search-index-body {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin: 100px;
+  margin-bottom: 40px;
+  padding-bottom: 90px;
+  margin-top: 50px;
+}
+#my-search-span {
+  color: #42bfbf;
+}
+#login,
+#signup,
+#password {
+  background-color: #000000;
+}
+#login-page,
+#signup-page,
+#password-page {
+  text-align: right;
+  justify-content: center;
+  margin-top: 15vh;
+  padding-top: 7vh;
+  z-index: 5;
+  font-size: 28px;
+  background: linear-gradient(45deg, #42bfbf 50%, #000000 50%);
+  color: #ffffff !important;
+  text-shadow: -2px -2px#42bfbf;
+}
+#login-h2,
+#signup-h2,
+#password-h2 {
+  font-size: 40px;
+}
+#login-remember {
+  font-size: 18px;
+}
+#login-links,
+#signup-links,
+#password-links {
+  color: #ffffff !important;
+  font-size: 18px;
+  text-shadow: none;
+  transition-duration: 0.3s;
+}
+#login-links:hover,
+#signup-links:hover,
+#password-links {
+  color: #000000 !important;
+}
+#signup-button {
+  font-size: 20px;
+}
+#password-button {
+  font-size: 16px;
+}
+.field {
+  margin-bottom: 5px;
 }
 #card1 {
   padding-bottom: 140px;
   min-height: 387px;
   width: 100%;
-  transform: rotate(20deg);
   margin-right: 50px;
 }
 #card2 {
+  margin-top: 25px !important;
   width: 100%;
-  transform: rotate(-20deg);
-  margin-left: 50px;
 }
 #card3 {
   margin-top: 25px !important;
-  min-width: 540px;
-  max-width: 540px;
   margin: 0 auto;
 }
+.about-us {
+  background-color: #000000;
+}
+.about-us-body {
+  background-color: #000000;
+}
+.about-title {
+  text-align: center;
+  padding-top: 25vh;
+  color: #d3d3cd;
+}
+@media screen and (max-width: 768px) {
+  .about-title {
+    padding-top: 20vh !important;
+  }
+  .about-us-card {
+    margin-top: 75px !important;
+    width: 100%;
+  }
+  #card1,
+  #card2,
+  #card3 {
+    margin-top: 0px;
+    margin-bottom: 0px;
+  }
+  #card1 {
+    margin-right: 0px;
+    max-width: 100%;
+  }
+  #card2 {
+    margin-left: 0px;
+    margin-top: 25px !important;
+  }
+  #card3 {
+    min-height: 387px;
+  }
+  .search-result-card,
+  #my-search-card {
+    transform: rotate(0deg) !important;
+  }
+  #search-results-page {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    margin: 30px;
+    margin-left: 0px;
+    margin-bottom: 0px;
+  }
+}
+
 .about-us-card {
-  border: 1px solid black;
+  border: 2px solid black;
   padding: 20px;
   border-radius: 20px;
-  background-color: bisque;
+  background-color: #42bfbf;
+}
+svg {
```

Comment:
Optional: this is a global change to all `<svg>`, when this change is probably intended for a set of them? Perhaps a class could be established, otherwise attempts to add more `<svg>` in the future will be confusing.

Reply:
Will icebox for now since we know we won't require any more svg's